dtfs-central-api/src/services/tfm/deal-cancellation.service.ts

```diff
@@ -114,11 +110,7 @@ export class DealCancellationService {
       dealType,
     });
 
-    await Promise.all(
-      riskExpiredFacilities.map(({ _id: facilityId }) =>
-        PortalFacilityService.updateStatus({ facilityId, status: FACILITY_STATUS.RISK_EXPIRED, dealType, auditDetails }),
-      ),
-    );
+    await PortalFacilityRepo.updateByDealId(dealId, { facilityStage: FACILITY_STAGE.RISK_EXPIRED }, auditDetails);
```

Comment:
I'm not sure about existing patterns off the top of my head - how about calling the method e.g `bulkUpdateByDealId` or `updatedMultipleByDealId`  instead `updateByDealId`? Currently it's not clear it's a bulk operation

Reply:
I've found a couple of examples `UtilisationReportRepo.findAllByBankId` & `TfmFacilitiesRepo.deleteManyByDealId` - I think i prefer the latter (since it keeps it consistent with mongodb methods)

Reply:
Makes sense, `updateManyByDealId`, nice 👍